src/Containers.jsx

```diff
@@ -545,6 +560,23 @@
         );
     }
 
+    copyContainer(container, containerDetail, localImages) {
+        this.context.show(<ImageRunModal user={this.props.user}
+                                    localImages={localImages}
+                                    pod={this.props.pods[container.Pod + container.isSystem]}
+                                    registries={this.props.registries}
+                                    selinuxAvailable={this.props.selinuxAvailable}
+                                    podmanRestartAvailable={this.props.podmanRestartAvailable}
+                                    userServiceAvailable={this.props.userServiceAvailable}
+                                    systemServiceAvailable={this.props.systemServiceAvailable}
+                                    onAddNotification={this.props.onAddNotification}
+                                    version={this.props.version}
+                                    container={container}
+                                    containerDetail={containerDetail}
+                                    prefill
```

Comment:
These 14 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230428-110256-01fd85db-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)